src/components/UserList.js

```diff
@@ -0,0 +1,69 @@
+import UserAddButton from './UserAddButton.js';
+import UserDeleteButton from './UserDeleteButton.js';
+import UserItem from './UserItem.js';
+import usersStore, { ADD_USER, DELETE_USER } from '../modules/users.js';
+import selectedUserStore, { GET_USER } from '../modules/selectedUser.js';
+
+const MINIMUN_USER_LENGTH = 2;
+
+const UserList = async () => {
+  const $userList = document.querySelector('#user-list');
+
+  const onAddUser = async ({ target }) => {
+    if (target.classList.contains('user-create-button')) {
+      const userName = prompt('이름을 입력해주세요');
+      if (userName === null) {
+        return;
+      }
+      if (userName.trim().length < MINIMUN_USER_LENGTH) {
+        alert('닉네임은 2자 이상 입력해야 합니다.');
```

Comment:
```suggestion
        alert(`닉네임은 ${MINIMUN_USER_LENGTH}자 이상 입력해야 합니다.`);
```

상수로 ```MINIMUN_USER_LENGTH```를 분리하고 조건 역시 ```MINIMUN_USER_LENGTH```를 사용해서 검증하는만큼 alert에도 ```MINIMUN_USER_LENGTH```를 넣어서 같이 관리하게 되면 좋을거 같습니다!

Reply:
오 좋은 아이디어 감사합니다~